on_startup.py

```diff
@@ -1,37 +1,69 @@
-import glob
-import json
-import logging
+import os
 import requests
 import subprocess
-from requests.exceptions import RequestException
 
 import django
+
+from requests.exceptions import RequestException
+
 from django.core.management import call_command
 
-logger = logging.getLogger(__name__)
+HIGLASS_DATA_TYPE = "higlass_data_type"
+HIGLASS_FILE_TYPE = "higlass_file_type"
+
+TILE_SETS = {}
+
+
+def get_refinery_input_as_json():
+    return requests.get(os.environ["INPUT_JSON_URL"]).json()
 
 
 def populate_higlass_data_directory(data_dir):
     """
-    Download remote files specified by urls in the input.json file
+    Download remote files specified by urls in the data provided by a GET to
+    the provided INPUT_JSON_URL
     :param data_dir: <String> Path to directory to populate with data
     """
-    with open("/data/input.json") as f:
-        config_data = json.loads(f.read())
+    config_data = get_refinery_input_as_json()
+
+    for refinery_node_uuid in config_data["node_info"]:
+        TILE_SETS[refinery_node_uuid] = {}
+
+        refinery_node = config_data["node_info"][refinery_node_uuid]
+        TILE_SETS[refinery_node_uuid]["file_url"] = refinery_node["file_url"]
+        TILE_SETS[refinery_node_uuid]["file_name"] = refinery_node[
+            "file_url"].split("/")[-1]
+
+        for key in refinery_node["node_solr_info"].keys():
+            if HIGLASS_FILE_TYPE in key.lower():
+                TILE_SETS[refinery_node_uuid][HIGLASS_FILE_TYPE] = (
+                    refinery_node["node_solr_info"][key]
+                )
+            if HIGLASS_DATA_TYPE in key.lower():
+                TILE_SETS[refinery_node_uuid][HIGLASS_DATA_TYPE] = (
+                    refinery_node["node_solr_info"][key]
+                )
 
-    for url in config_data["file_relationships"]:
         try:
             # Streaming GET for potentially large files
-            response = requests.get(url, stream=True)
+            response = requests.get(
+                TILE_SETS[refinery_node_uuid]["file_url"],
+                stream=True
+            )
         except RequestException as e:
             raise RuntimeError(
                 "Something went wrong while fetching file from {} : {}".format(
-                    url,
+                    TILE_SETS[refinery_node_uuid]["file_url"],
                     e
                 )
             )
         else:
-            with open('{}{}'.format(data_dir, url.split("/")[-1]), 'wb') as f:
+            with open(
+                '{}{}'.format(
+                    data_dir,
+                    TILE_SETS[refinery_node_uuid]["file_name"]
+                ), 'wb'
+            ) as f:
```

Comment:
Here and above, I think it was more clear to have a short variable instead of `TILE_SETS[refinery_node_uuid]["file_name"]` and `TILE_SETS[refinery_node_uuid]["file_url"]`